python/ray/tune/execution/tune_controller.py

```diff
@@ -320,18 +314,9 @@ def experiment_path(self) -> str:
 
     def _create_checkpoint_manager(self):
         return _ExperimentCheckpointManager(
-            storage=self._storage,
-            checkpoint_period=self._checkpoint_period,
-            sync_every_n_trial_checkpoints=self._trial_checkpoint_config.num_to_keep,
```

Comment:
I realize that I actually don't need to remove this. I thought that my change of removing the 5 minute `sync_period` was enough, but technically we still need to upload every time a trial checkpoint happens because otherwise you could resume from a stale state and start from scratch.